hack/after-build/run-codegen.sh

```diff
@@ -33,6 +34,7 @@ clientgen=$(kube::util::find-binary "client-gen")
 # update- and verify- scripts.
 ${clientgen} "$@"
 ${clientgen} -t "$@"
+${deepcopygen} "$@"
```

Comment:
Actually - the issue was that there was a fatal caused by generating deep-copy function for conversion.Equalities:
https://github.com/kubernetes/kubernetes/blob/master/pkg/conversion/deep_equal.go#L25

(the one that you asked about before). I solved it by adding this "temporary hack" that you asked about above.